reconcile/utils/repo_owners.py

```diff
@@ -133,7 +133,10 @@ def _get_owners_map(self):
         for owner_file in owner_files:
             raw_owners = self._git_cli.get_file(path=owner_file['path'],
                                                 ref=self._ref)
-            owners = yaml.safe_load(raw_owners.decode())
+            try:
+                owners = yaml.safe_load(raw_owners.decode())
+            except yaml.parser.ParserError:
```

Comment:
Is there another before this one that guarantees that the error mentioned in this PR is the only one that can happens? If not, maybe we should check the error message.